airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/cdc/MySqlCdcStateHandler.java

```diff
@@ -39,7 +39,7 @@ public boolean isCdcCheckpointEnabled() {
   public AirbyteMessage saveState(final Map<String, String> offset, final SchemaHistory<String> dbHistory) {
     final JsonNode asJson = serialize(offset, dbHistory);
 
-    LOGGER.info("debezium state: {}", asJson);
+    LOGGER.debug("debezium state: {}", asJson);
```

Comment:
Do we log something similar in any of the other connectors?

Reply:
yes (postgres and mongodb) - I'm not sure the value of this log line - I'm thinking we should validate if this change will fix the problem on mysql first?